.github/workflows/format-code.yml

```diff
@@ -9,25 +9,12 @@ jobs:
 
   format-code:
     name: Format Code
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Something's up with this diff as I assume this change came from `master`?

Reply:
I did merge master into this PR.